ng-dev/pr/common/checkout-pr.ts

```diff
@@ -109,6 +109,7 @@ export async function checkOutPullRequestLocally(
   try {
     // Fetch the branch at the commit of the PR, and check it out in a detached state.
     info(`Checking out PR #${prNumber} from ${fullHeadRef}`);
+    info(`To push to this PR branch: git push ${GITHUB_SSH_PREFIX}:${pr.headRef.repository.nameWithOwner} HEAD:${headRefName} -f`);
```

Comment:
I'm not sure it's generally correct to always use the SSH prefix?

Reply:
I think not everybody uses SSH, and it's opt-in in the `ng-dev` config. I think it would make sense to either print both, or go with just showing the fork branch user and name

Reply:
I updated it to print both URLs.